app/models/katello/content_view_package_filter_rule.rb

```diff
@@ -9,6 +9,19 @@ class ContentViewPackageFilterRule < Katello::Model
 
     validates_lengths_from_database
     validates :name, :presence => true
+    validate :ensure_unique_attributes
     validates_with Validators::ContentViewFilterVersionValidator
+
+    def ensure_unique_attributes
+      other = self.class.where(:name => self.name, 
+                               :version => self.version, 
+                               :content_view_filter_id => self.content_view_filter_id,
+                               :min_version => self.min_version,
+                               :max_version => self.max_version)
+      other.not(:id => self.id) if self.id
```

Comment:
I don't think this will work, as you aren't saving the value of the other.not here.